### PR TITLE
DO NOT MERGE: test dhcp activity

### PR DESCRIPTION
**Pull request by vladikr**
Signed-off-by: Vladik Romanovsky <vromanso@redhat.com>

### CODE REVIEW

**Comment by vladikr** [comment]
retest this please

**Comment by vladikr** [comment]
retest this please

File: tests/vm_networking_test.go [hunk 1]
```diff
@@ -105,6 +105,18 @@ var _ = Describe("Networking", func() {
 			vm, err = virtClient.VM(tests.NamespaceTestDefault).Get(vm.ObjectMeta.Name, v13.GetOptions{})
 			Expect(err).ToNot(HaveOccurred())
 
+			dhcpAddr := vm.Status.Interfaces[0].IP
+
+			expecterDhcp, _, err := tests.NewConsoleExpecter(virtClient, vm, 10*time.Second)
+			defer expecterDhcp.Close()
+			Expect(err).ToNot(HaveOccurred())
+			resp1, err := expecterDhcp.ExpectBatch([]expect.Batcher{
+				&expect.BExp{R: "Sending discover..."},
```
Comment (rmohr) [inline tests/vm_networking_test.go]: At the end  the vms always booted, so I don't think that his helps you. What I would love to see, is that we log and timestamp every line comming from the console. We have access to everything coming from the vm, before it passed to the Expecter here: https://github.com/kubevirt/kubevirt/blob/master/tests/utils.go#L951. I would simply log every line coming in and then move it forward. If the test fails, the logs will get printed.

**Comment by rmohr** [comment]
Closing this since we found the issue for #936

**Pull request closed without merging**